.travis.yml

```
@@ -34,6 +34,9 @@ matrix:
     env: SKIP_RELEASE=true
   - jdk: openjdk11
     env: SKIP_RELEASE=true MOCK_MAKER=mock-maker-inline
+  - jdk: openjdk8
+    script: ./check_reproducibility.sh
```

```suggestion
  - name: "Check reproducibility" 
    jdk: openjdk8
    script: ./check_reproducibility.sh
```

Taking a look in Travis right now, it looks like "just" another Jdk 8 build and this would allow people to identify this particular job at a glance if it starts failing.

---

I am going to do this in a follow-up, as we can do some more cleanup of the Travis tasks. Thanks for the suggestion 😄